can/interfaces/ixxat/canlib_vcinpl2.py

```diff
@@ -0,0 +1,948 @@
+    _canlib.map_symbol(
+        "canSchedulerStopMessage",
+        ctypes.c_long,
+        (HANDLE, ctypes.c_uint32),
+        __check_status,
+    )
+    _canlib.vciInitialize()
+except AttributeError:
+    # In case _canlib == None meaning we're not on win32/no lib found
+    pass
+except Exception as e:
+    log.warning("Could not initialize IXXAT VCI library: %s", e)
+# ---------------------------------------------------------------------------
+
+
+CAN_INFO_MESSAGES = {
+    constants.CAN_INFO_START: "CAN started",
+    constants.CAN_INFO_STOP: "CAN stopped",
+    constants.CAN_INFO_RESET: "CAN reset",
+}
+
+CAN_ERROR_MESSAGES = {
+    constants.CAN_ERROR_STUFF: "CAN bit stuff error",
+    constants.CAN_ERROR_FORM: "CAN form error",
+    constants.CAN_ERROR_ACK: "CAN acknowledgment error",
+    constants.CAN_ERROR_BIT: "CAN bit error",
+    constants.CAN_ERROR_CRC: "CAN CRC error",
+    constants.CAN_ERROR_OTHER: "Other (unknown) CAN error",
+}
+
+CAN_STATUS_FLAGS = {
+    constants.CAN_STATUS_TXPEND: "transmission pending",
+    constants.CAN_STATUS_OVRRUN: "data overrun occurred",
+    constants.CAN_STATUS_ERRLIM: "error warning limit exceeded",
+    constants.CAN_STATUS_BUSOFF: "bus off",
+    constants.CAN_STATUS_ININIT: "init mode active",
+    constants.CAN_STATUS_BUSCERR: "bus coupling error",
+}
+# ----------------------------------------------------------------------------
+
+
+class IXXATBus(BusABC):
+    """The CAN Bus implemented for the IXXAT interface.
+
+    .. warning::
+
+        This interface does implement efficient filtering of messages, but
+        the filters have to be set in :meth:`~can.interfaces.ixxat.IXXATBus.__init__`
+        using the ``can_filters`` parameter. Using :meth:`~can.interfaces.ixxat.IXXATBus.set_filters`
+        does not work.
+
+    """
+
+    def __init__(self, channel, can_filters=None, **kwargs):
+        """
+        :param int channel:
+            The Channel id to create this bus with.
+
+        :param list can_filters:
+            See :meth:`can.BusABC.set_filters`.
+
+        :param bool receive_own_messages:
+            Enable self-reception of sent messages.
+
+        :param int UniqueHardwareId:
+            UniqueHardwareId to connect (optional, will use the first found if not supplied)
+
+        :param int bitrate:
+            Channel bitrate in bit/s
+
+        :param int data_bitrate:
+            Channel bitrate in bit/s (only in CAN-Fd if baudrate switch enabled).
+
+        :param int extended:
+            Default False, enables the capability to use extended IDs.
+        """
+        if _canlib is None:
+            raise CanInterfaceNotImplementedError(
+                "The IXXAT VCI library has not been initialized. Check the logs for more details."
+            )
+        log.info("CAN Filters: %s", can_filters)
+        log.info("Got configuration of: %s", kwargs)
+        # Configuration options
```

Comment:
Changed only keyword arguments, but I left the parameter names unchanged (no snake_case). I'm worried in case changing them breaks users code, but if you are convinced, I don't mind to change it after you feedback.

Reply:
Interesting, thanks, changed.